<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StrackDesktop</class>
 <widget class="QDialog" name="StrackDesktop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>803</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Strack Connection</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../images/icons.qrc">
    <normaloff>:/icons/strack.png</normaloff>:/icons/strack.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background-color: rgb(255, 255, 255);
}</string>
  </property>
  <layout class="QVBoxLayout" name="win_layout">
   <property name="spacing">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="main_frame">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <layout class="QVBoxLayout" name="main_layout" stretch="1,2,20,1">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="top_grp">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>146</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>146</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox#top_grp
{
	background-image: url(:/backgroud/top_back.png);
	padding:0;
	margin:0;
	border-style:soild;
	border-width: 0px;
	border-top-left-radius:2px;
	border-top-right-radius:2px;
	border-bottom-right-radius:0px;
	border-bottom-left-radius:0px;

}</string>
        </property>
        <property name="title">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="top_group_layout" stretch="0,0">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="title_bar_grp">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>26</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QGroupBox{
	border:none;
	padding:3px;
	margin:0;
}</string>
           </property>
           <layout class="QHBoxLayout" name="titile_bar_layout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>5</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="icon_label">
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../../images/icons.qrc">:/icons/strack.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="title_label">
              <property name="styleSheet">
               <string notr="true">font: 75 10pt &quot;Leelawadee UI&quot;;
color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string> Strack-Connection</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QToolButton" name="minimize_btn">
              <property name="minimumSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QToolButton#minimize_btn
{
	image: url(:/icons/min_white.png);
	height: 20px;
	width: 20px;
	border: none;
}
QToolButton#minimize_btn::hover
{
	image: url(:/icons/min_white.png);
	background-color: rgba(135, 158, 199, 100);;
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>10</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QToolButton" name="close_btn">
              <property name="minimumSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QToolButton#close_btn
{
	image: url(:/icons/close_white.png);
	height: 20px;
	width: 20px;
	border: none;
}
QToolButton#close_btn::hover
{
	image: url(:/icons/close_white.png);
	background-color: #D50000;
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="user_info_grp">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>120</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>120</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QGroupBox{
	border:none;
	padding:0;
	margin:0;
}</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <layout class="QHBoxLayout" name="user_info_layout" stretch="0,0,0,2,0">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <spacer name="horizontalSpacer_8">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="thubnail_label">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../../images/other_images.qrc">:/thumbnails/default_avatar.png</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>10</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label">
                <property name="styleSheet">
                 <string notr="true">font: 75 12pt &quot;Leelawadee UI&quot;;
color: rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>Aaron</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="tab_group">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #E0E0E0;
padding:3px;
border-color:#E7EFF8;
border-width:1px 0px 1px 0px;
border-style:solid;</string>
        </property>
        <layout class="QHBoxLayout" name="tab_layout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="action_btn">
           <property name="styleSheet">
            <string notr="true">QToolButton#action_btn
{
	image: url(:/icons/action_gray.png);
	height: 20px;
	border: none;
}
QToolButton#action_btn::hover
{
	image: url(:/icons/action_blue.png);
}
QToolButton#action_btn:checked
{
	image: url(:/icons/action_dark_blue.png);
}
</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="task_btn">
           <property name="styleSheet">
            <string notr="true">QToolButton#task_btn
{
	image: url(:/icons/task_gray.png);
	height: 20px;
	border: none;
}
QToolButton#task_btn::hover
{
	image: url(:/icons/task_blue.png);
}
QToolButton#task_btn::checked
{
	image: url(:/icons/task_dark_blue.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="publish_btn">
           <property name="styleSheet">
            <string notr="true">QToolButton#publish_btn
{
	image: url(:/icons/publish_gray.png);
	height: 20px;
	border: none;
}
QToolButton#publish_btn::hover
{
	image: url(:/icons/publish_blue.png);
}
QToolButton#publish_btn::checked
{
	image: url(:/icons/publish_dark_blue.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="chat_btn">
           <property name="styleSheet">
            <string notr="true">QToolButton#chat_btn
{
	image: url(:/icons/chat_gray.png);
	height: 20px;
	border: none;
}
QToolButton#chat_btn::hover
{
	image: url(:/icons/chat_blue.png);
}
QToolButton#chat_btn::checked
{
	image: url(:/icons/chat_dark_blue.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="work_area_grp">
        <property name="styleSheet">
         <string notr="true">background-color: #ffffff;</string>
        </property>
        <property name="title">
         <string/>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="workarea_layout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QStackedWidget" name="work_stack">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="ActionsUI" name="action_stack"/>
           <widget class="MyTasksUI" name="mytask_stack"/>
           <widget class="PublishUI" name="publish_stack"/>
           <widget class="ChatUI" name="chat_stack"/>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="buttom_grp">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #E0E0E0;
padding:0px;
border-color:#E7EFF8;
border-width:1px 0px 0px 0px;
border-style:solid;
border-top-left-radius:0px;
border-top-right-radius:0px;
border-bottom-right-radius:2px;
border-bottom-left-radius:2px;</string>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="menu_layout" stretch="0,0,0">
           <item>
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>10</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QToolButton" name="menu_btn">
             <property name="styleSheet">
              <string notr="true">QToolButton#menu_btn
{
	image: url(:/icons/menu_blue.png);
	height: 20px;
	border: none;
}
QToolButton#menu_btn::checked
{
	image: url(:/icons/menu_dark_blue.png);
}
</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="sizegrip_layout">
           <item>
            <spacer name="size_spacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>10</width>
               <height>58</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ActionsUI</class>
   <extends>QWidget</extends>
   <header>actions_ui/ActionsUI</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MyTasksUI</class>
   <extends>QWidget</extends>
   <header>mytasks_ui/MyTasksUI</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PublishUI</class>
   <extends>QWidget</extends>
   <header>publish_ui/PublishUI</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ChatUI</class>
   <extends>QWidget</extends>
   <header>chat_ui</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../images/icons.qrc"/>
  <include location="../../images/other_images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>close_btn</sender>
   <signal>clicked()</signal>
   <receiver>StrackDesktop</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>-11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minimize_btn</sender>
   <signal>clicked()</signal>
   <receiver>StrackDesktop</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
